packages/react/src/components/Message/MessageHeader.js

```diff
@@ -83,9 +82,9 @@ const MessageHeader = ({ message, isTimeStamped = true }) => {
     }
   };
 
-  const userRoles = roles[message.u.username]
-    ? roles[message.u.username].roles
-    : null;
+  // const userRoles = roles[message.u.username]
```

Comment:
why comment? can we remove this or is it required?

Reply:
I think this was left by Abhinav to implement something later on, so I just commented it out since I was getting linting issues before committing.

Was not sure why it was there so I commented it out instead of removing.
<img width="637" alt="Screenshot 2024-03-10 212112" src="https://github.com/RocketChat/EmbeddedChat/assets/95426993/05613670-1e8c-4142-ba24-0c9d01ac9644">